app/code/Magento/Backend/view/adminhtml/layout/adminhtml_cron_block.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0"?>
```

Comment:
It is suggested to use UI Components for the grids in the new code, other approaches are considered legacy.

Reply:
Sure.  Have you got an example I can look at?

Reply:
You may start with the [DevDocs article](http://devdocs.magento.com/guides/v2.2/ui_comp_guide/components/ui-listing-grid.html) on this subject, however syntax in the article is slightly outdated. I consider these examples, from MSI project, pretty well implemented: [Component declaration](https://github.com/magento-engcom/magento2/blob/develop/app/code/Magento/Inventory/view/adminhtml/ui_component/inventory_source_listing.xml) + [DataProvider for the Component](https://github.com/magento-engcom/magento2/blob/develop/app/code/Magento/Inventory/Ui/DataProvider/SourceDataProvider.php).
Please let us know if you have any more questions regarding this.